feature-libs/asm/components/asm-bind-cart/asm-bind-cart.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+import { AsmFacade } from '@spartacus/asm/root';
+import { ActiveCartFacade, MultiCartFacade } from '@spartacus/cart/base/root';
+import {
+  GlobalMessageService,
+  GlobalMessageType,
+  OCC_CART_ID_CURRENT,
+  User,
+} from '@spartacus/core';
+import { Observable, Subscription } from 'rxjs';
+
+@Component({
+  selector: 'cx-asm-bind-cart',
+  templateUrl: './asm-bind-cart.component.html',
+})
+export class AsmBindCartComponent implements OnInit, OnDestroy {
+  @Input() customer: User;
+  @Input() cartId: FormControl = new FormControl('', [
+    Validators.required,
+    Validators.minLength(1),
+  ]);
+  isCustomerEmulationSessionInProgress$: Observable<boolean>;
+
+  protected subscription = new Subscription();
+
+  constructor(
+    protected globalMessageService: GlobalMessageService,
+    protected asmFacadeService: AsmFacade,
+    protected activeCartFacade: ActiveCartFacade,
+    protected multiCartFacade: MultiCartFacade
+  ) {}
+
+  ngOnInit(): void {
+    this.subscription.add(
```

Comment:
See other comment and discussion thread about setting the active cart id in the input field.

Reply:
I did not attend the discussion. But, only anonymous cart can be bound to customer. So, why we set user's active cart here?

Reply:
@WeizhengSap you are right, only an anonymous cart can be bound to a customer.  Moreover, the active cart ID is already available on the cart page.

The discussion I referred to is [here](https://github.com/SAP/spartacus/pull/15776#discussion_r893863621) on the same PR.
In the end, it appears that this was the behaviour of the ACC UI and the PO decision is to port the behaviour over to Spartacus to achieve feature parity.